book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,555 @@
+    * Consider digital accessibility on social media and in job adverts
+      Not all advertising venues are digitally accessible (See "Resources" for more reading)
+    * Leverage your networks, as well as those of others in the organisation, to share the opportunity advertised for example, Slack, LinkedIn, word of mouth at conferences 
+        * When utilising your networks, consider if there are any groups potentially missing. Consider using alternative spaces (examples listed in Resources section) to reach groups that aren’t in your network.
+      *If including images in the job advert or description, ensure alternative text is included (See "Resources" for more reading)
+         *Any links to other websites or pages should be linked in the text and not in images
+- [ ] Set a suitable closing deadline
+    * Ensure a closing date is included on the advert
+        * Advertise roles minimally for 2-4 weeks where possible
+    * Where possible, stick to the advertised close date. Avoid stating that the role will be closed early if a suitable candidate is identified.  
+    * Consider the day of the week and time of closing your advert. For example, a closing date set on a weekend may be valuable for people who have limited free time.
+    * Be explicit on the time zone which will be used to close the application period 
+
+
+
+### Before job advert closes
+
+
+
+- [ ] Book time(s) for interview panel *[Contributing to this action may be out of scope for hiring managers depending on organisational processes]*
+    * Consider flexibility and other requirements for applicants AND members of the interview panel.
+  For example, ensuring that panel members have sufficient  breaks and that flexibility in scheduling can be offered to applicants - planning should be undertaken in partnership with your HR team.  
+    * Ensure there is a diverse interview panel for every interview - consider gender, race, diversity of thought, interest/impacted group perspective. 
+    * Consider having the option of both in-person or remote interviews for candidates to select from
+        - [ ] Consider providing reimbursement for expenses incurred for candidates attending in-person interviews 
+- [ ] Decide on assessments and criteria
+    * Be clear on what criteria is being used for shortlisting
+    * Ensure assessment criteria focus on the person specification within the job description. This should help mitigate bias during hiring decisions later on.
+
+
+### At the deadline 
+
+
+
+- [ ] Close job advert 
+    * Be transparent with the status of the application and timelines
+    * Ensure reasonable timeframes are kept when making decisions 
+    * Either at this stage, or at the end of the process, review anonymised demographic data of candidates to see if your advertising avenues were ‘effective’, how ever this may be determined by your organisation
+
+
+### 
+
+
+## Resources
+
+
+### Further reading 
```

Comment:
If we can, it would be good to add these to the Bibliography of the overall Turing Way :)